doc/prometheus_mnesia_collector.md

```diff
@@ -50,6 +50,9 @@ Number of transactions logged.
 Type: counter.<br />
 Total number of transaction restarts.
 
+* `erlang_mnesia_memory_usage_bytes`<br/>
+Type: counter.<br/>
```

Comment:
I believe this is a [gauge](https://prometheus.io/docs/concepts/metric_types/#gauge), not a [counter](https://prometheus.io/docs/concepts/metric_types/#counter). If you agree, can you please update it?

Happy to QA & merge afterwards 😉

Reply:
You are absolutely right, a gauge is more suitable.